conf/messages

```diff
@@ -8,14 +8,14 @@ enrolCurrentUser.currentUserId = My current user ID {0}
 enrolCurrentUser.otherUserId = Another user ID
 enrolCurrentUserId.error.required = Confirm which Government Gateway user ID you want to use
 enrolCurrentUser.warning = You have access to Self Assessment under user ID {0}. We recommend signing back in with this user ID so that you can access Self Assessment from your personal tax account.
-landingPage.title = Only your current user ID can access your personal tax account from now on
-landingPage.heading = Only your current user ID can access your personal tax account from now on
-landingPage.paragraph1 = You are currently signed in with the Government Gateway user ID {0}.
-landingPage.paragraph2 = This is now the only user ID that can access your personal tax account. Please keep the details of this user ID safe.
-landingPage.paragraphSA = You also have access to Self Assessment with your current user ID.
-landingPage.heading2 = What has happened to my other Government Gateway user IDs?
-landingPage.paragraph3 = If you have any other Government Gateway user IDs, they will lose access to your personal tax account.
-landingPage.continue = Continue
+enrolledForPT.title = Only your current user ID can access your personal tax account from now on
+enrolledForPT.heading = Only your current user ID can access your personal tax account from now on
+enrolledForPT.paragraph1 = You are currently signed in with the Government Gateway user ID {0}.
```

Comment:
This should be "You are currently signed in with the Government Gateway user ID **ending with** {0}.

Reply:
This is a change from the AccountDetails model. We are having a separate story to cover it.

Reply:
The model itself will say ending with

Reply:
Either that or we have to update it in all the views